src/Organizations/OrganizationDetails/LinkedAgreements/LinkedAgreements.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Accordion,
+  Loading,
+  MultiColumnList,
+  NoValue,
+  TextLink,
+} from '@folio/stripes/components';
+import {
+  FolioFormattedDate,
+  PrevNextPagination,
+} from '@folio/stripes-acq-components';
+
+import { useLinkedAgreements } from '../../../common/hooks';
+import {
+  AGREEMENTS_LIST_COLUMNS,
+  AGREEMENTS_LIST_COLUMNS_MAPPING,
+  AGREEMENTS_LIST_VISIBLE_COLUMNS,
+} from './constants';
+
+const RESULT_COUNT_INCREMENT = 30;
+
+const formatter = {
+  [AGREEMENTS_LIST_COLUMNS.name]: agreement => (
+    <TextLink to={`/erm/agreements/${agreement.id}`}>
+      {agreement.name}
+    </TextLink>
+  ),
+  [AGREEMENTS_LIST_COLUMNS.startDate]: agreement => <FolioFormattedDate value={agreement.startDate} />,
+  [AGREEMENTS_LIST_COLUMNS.endDate]: agreement => <FolioFormattedDate value={agreement.endDate} />,
+  [AGREEMENTS_LIST_COLUMNS.agreementStatus]: agreement => agreement.agreementStatus?.label || <NoValue />,
```

Comment:
Do we know the list of agreement statuses? Maybe it's a good idea to make them translatable?

Reply:
Yep, probably it makes sense, thanks!)